.travis.yml

```diff
@@ -4,29 +4,50 @@ matrix:
   include:
     # Linux Python 2
     - sudo: required
+      if: branch=master
       language: python
       python: 2.7
       services: docker
       env: PYTHON=python
 
     # Linux Python 3
     - sudo: required
+      if: branch=master
       language: python
       python: 3.4
       services: docker
       env: PYTHON=python
 
     # macOS Python 2
     - os: osx
+      if: branch=master
       env: PYTHON=python2
 
     # macOS Python 3
     - os: osx
+      if: branch=master
       env: PYTHON=python3
       before_install:
         - brew update
         - brew outdated python || brew upgrade python
 
+    - os: windows
+      language: shell
+      before_install:
+       - choco install python3 --version 3.5.4 --no-progress --force -y --allowmultiple --override --installargs "'/quiet  InstallAllUsers=1 TargetDir=C:\Python35-x64'"
```

Comment:
Just for future reference, is there a reason we need all of these different arguments, here?
https://docs.travis-ci.com/user/languages/python/ seems to be suggesting to just `choco install python` (or maybe `python3`, I guess) and use the `python` executable afterwards.

I understand we need more flags/control for the different versions, but for the one that's just running `cibuildwheel` in the examples, it would be great to keep things simple.

Reply:
`--no-progress` means not show progress of download. In other cases the 4MB limit of log on travis may run out. 
`-y` accept all questions during installation.  
`-allowmultiple` allow multiple versions of package. Not sure if need here. Needed with next installations
`--override --installargs "'/quiet  InstallAllUsers=1 TargetDir=C:\Python35-x64'"` - set install directory